config.js

```diff
@@ -7,5 +7,5 @@ export const config = {
     DISCORD_TOKEN: process.env.DISCORD_TOKEN || undefined,
     PUBLIC_KEY: process.env.PUBLIC_KEY || undefined,
     LEETCODE_CHALLENGES_CHANNEL: process.env.LEETCODE_CHALLENGES_CHANNEL || undefined,
-    DEV_PRIMOBOT_CHANNEL: process.env.DEV_PRIMOBOT_CHANNEL || undefined,
+    STATS_CHANNEL: process.env.STATS_CHANNEL || undefined,
   };
```

Comment:
nit: it's a good practice to leave a new line at the end of the file.

Reply:
hmm, we should probably have a linter for this, bc, you know, ppl forget to add that empty space, and I don't think this is a real thread to JS, but I agree with the concept

Reply:
I just enabled this in VS Code - it'll be automagically added next time.

<img width="2156" alt="Screen Shot 2022-10-08 at 1 14 50 PM" src="https://user-images.githubusercontent.com/1899538/194721758-7f1c6e9d-0e4a-40a1-9742-d33ebd3b014f.png">